packages/orchestrator-ui-components/src/pages/processes/WfoStartProcessPage.tsx

```diff
@@ -79,22 +84,42 @@ export const WfoStartProcessPage = ({
     const [form, setForm] = useState<UserInputForm>({});
     const { productId, subscriptionId } = router.query as StartProcessPageQuery;
 
-    const startWorkflowPayload = useMemo(
-        () => getInitialWorkflowPayload({ productId, subscriptionId }),
+    const startProcessPayload = useMemo(
+        () => getInitialProcessPayload({ productId, subscriptionId }),
         [productId, subscriptionId],
     );
 
     const { stepUserInput, hasNext } = form;
     const { getStepHeaderStyle, stepListContentBoldTextStyle } =
         getStyles(theme);
 
+    const { data, isFetched } = useQueryWithGraphql(
+        GET_PROCESS_STEPS_GRAPHQL_QUERY,
+        {
+            processName,
+        },
+        `processSteps={processName}`,
+    );
+
+    const timeLineItems: TimelineItem[] =
```

Comment:
Might be nice to early return a `<Loading>` component before this statement to prevent data being possibly undefined from here onwards. See SubscriptionList page/component.

`data?.workflows?.page[0]?.steps && isFetched ? data.workflows.page[0].steps.map( ....` could be simplified